src/tsickle.ts

```diff
@@ -50,8 +51,43 @@ export interface AnnotatorOptions {
 
 export enum AnnotatorFeatures {
   LowerDecorators = 1 << 0,
+  /**
+   * filter out types when expanding `export * from ...`
+   */
+  FilterTypesInExportStart = 1 << 1,
+  /**
+   * emit enums as
```

Comment:
Can you comment here why you'd want to pass this flag or not?  I recall one of these is a temporary thing that we intend to remove.  I see below you have the "transformers need: ..." block, but is the intent that we remove all of these flags?